src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -351,8 +351,9 @@ def energyStorage(dict_values, group):
 
             # check if parameters are provided as timeseries
             for parameter in [EFFICIENCY, SOC_MIN, SOC_MAX]:
-                if parameter in dict_values[group][asset][subasset] and isinstance(
-                    dict_values[group][asset][subasset][parameter][VALUE], dict
+                if parameter in dict_values[group][asset][subasset] and (
+                    FILENAME in dict_values[group][asset][subasset][parameter]
+                    and HEADER in dict_values[group][asset][subasset][parameter]
```

Comment:
Usually, I would request a pytest to be created for this case. As this feature is covered with the benchmark test I created, and C0 functions are anyway lacking pytests, this is fine.